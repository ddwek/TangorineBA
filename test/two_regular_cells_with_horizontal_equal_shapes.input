seed = 1736261775
set_immutable_cells(): 4, 6, 13, 20, 29, 33, 
set_constraints(): 
3, BOTTOM, =
10, LEFT, x
12, TOP, x
14, RIGHT, x
22, RIGHT, =
28, RIGHT, =
29, TOP, x
33, LEFT, x
