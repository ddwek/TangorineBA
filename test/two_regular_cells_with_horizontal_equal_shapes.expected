n_steps = 38
steps: 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26, 27, 28, 29, 30, 31, 32, 33, 34, 35, 36, 37, 38,
[01;40;33m*[00m _[00m _[00m _[00m [01;44;37mD[00m _[00m 
[00;43;30m*[00m _[00m _[00m _[00m _[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m _[00m _[00m [01;44;37mD[00m _[00m 
[00;43;30m*[00m _[00m _[00m _[00m _[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m _[00m [01;44;37mD[00m _[00m 
[00;43;30m*[00m _[00m _[00m _[00m _[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m _[00m 
[00;43;30m*[00m _[00m _[00m _[00m _[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m _[00m 
[00;43;30m*[00m _[00m _[00m _[00m _[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m _[00m _[00m _[00m _[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m _[00m _[00m _[00m _[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m _[00m _[00m _[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m _[00m _[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m _[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m _[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
_[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m _[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m _[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m _[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m _[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
_[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m _[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m _[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m _[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m _[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
_[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m _[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m _[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m _[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m _[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [00;43;30m*[00m 
_[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [00;43;30m*[00m 
[01;40;37mD[00m _[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [00;43;30m*[00m 
[01;40;37mD[00m [01;40;37mD[00m _[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [00;43;30m*[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [00;43;30m*[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m _[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [00;43;30m*[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m _[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [00;43;30m*[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;33m*[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;31mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;31m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;31mD[00m 
[01;40;31m*[00m [01;40;31m*[00m [01;47;31mD[00m [01;40;31m*[00m [01;40;31mD[00m [01;40;31m*[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;47;31m*[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;31m*[00m 
 
[01;40;33m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;37mD[00m 
[00;43;30m*[00m [01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m 
[01;40;37mD[00m [00;43;30m*[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m 
[01;40;33m*[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;37mD[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;40;37mD[00m [01;40;33m*[00m [00;43;30m*[00m 
[01;40;37mD[00m [01;40;37mD[00m [01;40;33m*[00m [01;44;37mD[00m [01;40;33m*[00m [01;40;33m*[00m 
 
